tools/crm_resource_runtime.c

```diff
@@ -1790,7 +1795,14 @@ cli_resource_execute_from_params(pcmk__output_t *out, const char *rsc_name,
             goto done;
 
         if (op->stdout_data || op->stderr_data) {
-            out->subprocess_output(out, op->rc, op->stdout_data, op->stderr_data);
+            xmlNodePtr doc = string2xml(op->stdout_data);
```

Comment:
We could potentially also change this to validate op->stdout_data against the yet-to-be-decided resource agent XML schema.

Reply:
I'd rather not introduce a dependency on that schema, especially since it won't be standardized for a while